docs/infra/digitalocean.md

```diff
@@ -0,0 +1,90 @@
+# PASS Digital Ocean Deployment
+
+This will document how to run PASS in [Digital Ocean](https://www.digitalocean.com)
+as well as how to use [Digital Ocean](https://www.digitalocean.com) as a droplet
```

Comment:
Instead of "how to use Digital Ocean as a droplet as Self-Hosted GitHub Runners" it should probably be "how to use Digital Ocean droplets as Self-Hosted GitHub Runners"